src/pages/login/components/LoginForm.tsx

```diff
@@ -2,85 +2,85 @@ import { Button } from "@/components/ui/button";
-              autoCorrect="off"
-            />
+        <div className="grid gap-6">
+          <div className="grid gap-5">
+            <div className="grid gap-1.5">
+              <Label htmlFor="email">
+                Email / Phone Number
+              </Label>
+              <Input
+                id="email"
+                placeholder="Enter your email / phone number"
+                type="text"
+                autoCapitalize="none"
+                autoComplete="off"
+                autoCorrect="off"
+                className="border-0 border-b border-b-grey-500 rounded-none py-2.5 px-0 text-base"
+                required
+              />
+            </div>
+            <div className="grid gap-1">
+              <Label htmlFor="password">
+                Password
+              </Label>
+              <Input
+                id="password"
+                placeholder="Enter your password"
+                type="password"
+                autoCapitalize="none"
+                autoComplete="off"
+                autoCorrect="off"
+                className="border-0 border-b border-b-grey-500 rounded-none py-2.5 px-0 text-base"
+                required
+              />
+            </div>
           </div>
-          <Button type="submit">
+          <Link to={'/forgot-password'} className="text-right text-gray-400 text-sm text-medium">
+            Forgot Password
+          </Link>
+          <Button type="submit" className="py-4 bg-primary-500 text-white rounded-xl">
             {isPending && <Loader className="mr-2 h-4 w-4 animate-spin" />}
-            Sign In with Email
+            Sign In 
           </Button>
         </div>
       </form>
-      <div className="relative">
-        <div className="relative flex justify-center text-xs uppercase">
-          <span className="bg-background text-muted-foreground px-2">
-            Or continue with
-          </span>
-        </div>
-      </div>
-      <div className="flex justify-between">
-        <Button variant="outline" type="button" disabled={isPending}>
-          {isPending ? (
-            <Loader className="mr-2 h-4 w-4 animate-spin" />
-          ) : (
-            <Chrome className="mr-2 h-4 w-4" />
-          )}
-          Google
-        </Button>
-        <Button variant="outline" type="button" disabled={isPending}>
-          {isPending ? (
-            <Loader className="mr-2 h-4 w-4 animate-spin" />
-          ) : (
-            <Github className="mr-2 h-4 w-4" />
-          )}
-          Github
-        </Button>
-        <Button variant="outline" type="button" disabled={isPending}>
+      <div className="grid">
+        <Button variant="outline" type="button" disabled={isPending} className="rounded-xl border-gray-200 gap-2">
           {isPending ? (
             <Loader className="mr-2 h-4 w-4 animate-spin" />
           ) : (
-            <Facebook className="mr-2 h-4 w-4" />
+            <img src="./src/assets/google.svg" />
```

Comment:
buat Image dibuat kaya mas Saiful gini ya mba, biar ke load di website. Soalnya kalo pake src ga kebaca.

https://github.com/SYNRGY-6-TEAM-5/frontend-web/blob/840c99f25837fb56d00bd105387569616eb0d81f/src/components/containers/HeroSection.tsx#L41

![image](https://github.com/SYNRGY-6-TEAM-5/frontend-web/assets/68576415/70a883ed-7e30-493f-bbab-26f7e735cd42)

Reply:
Atau mungkin biar ga banyak banyak asset static nya untuk icon simple kyk google, show password bisa pakai [phosporus-icon](https://phosphoricons.com/)

Makainya gini

```suggestion
import { GoogleLogo  } from "@phosphor-icons/react";

<GoogleLogo size={24} weight="bold" />
```